pnpm/tsconfig.json

```diff
@@ -4,7 +4,6 @@
     "composite": false,
     "declaration": false,
     "outDir": "lib",
-    "sourceMap": false,
```

Comment:
This now uses the inherited field from:

https://github.com/pnpm/pnpm/blob/fix-pnpm-test-source-map-misalignment/node_modules/@pnpm/tsconfig/tsconfig.json#L13